ComponentKit/TransactionalDataSources/Common/CKTransactionalComponentDataSource.mm

```diff
@@ -260,17 +260,14 @@ static void verifyChangeset(CKTransactionalComponentDataSourceChangeset *changes
                             CKTransactionalComponentDataSourceState *state,
                             NSArray<id<CKTransactionalComponentDataSourceStateModifying>> *pendingAsynchronousModifications)
 {
-#if CK_ASSERTIONS_ENABLED
```

Comment:
I'm pretty OK with doing this all the time. Let's measure the impact on performance and see if it's a problem.